server.js

```diff
@@ -1,24 +1,23 @@
 const bodyParser = require('body-parser');
 const path = require('path');
 const express = require('express');
-const Stock = require('./database-mysqldb/Stocks.js');
-const Account = require('./database-mysqldb/Accounts.js');
+const controller = require('./controller');
```

Comment:
Nice job with MVC architecture. Even though it seems excessive, this will be useful for more complex programs. https://en.wikipedia.org/wiki/Separation_of_concerns